cmd/nerdctl/login_linux_test.go

```diff
@@ -75,3 +76,102 @@ func TestLoginWithSpecificRegHosts(t *testing.T) {
 	}
 
 }
+
+func TestLoginWithPlainHttp(t *testing.T) {
```

Comment:
I'm not sure if this test works, it seems that `registry:2` image can't verify this issue.

I tested it locally through nexus3